src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -138,6 +165,14 @@ const GordonSchedulePanel = (props) => {
     </div>
   );
 
+  // We had to make the meeting day array in to a string with commas because the add-to-calendar
+  // required a string of weekdays for setting the recurring days
+
+  const meetingDayArray = selectedCourseInfo?.meetingDays;
+  const recurringDays = meetingDayArray?.map((day) => `${day}`).join(', ');
+  const courseTitle = selectedCourseInfo?.title.split('in')[0];
+  const courseLocation = selectedCourseInfo?.title.split('in')[1];
```

Comment:
this is not particularly good practice. Why is this not handled when courses are selected?